obfuscated/jasonzhao6/submission-2.js

```diff
@@ -0,0 +1,11 @@
+// Copy paste into Chrome's v69 browser console.
+
+[[
+    'Jan','Feb','Mar','Apr','May','Jun',
+    'Jul','Aug','Sep','Oct','Nov','Dec'
+
+  ].slice(6,11).map(e=>e[0]),' ',
```

Comment:
Excellent! Straight from [_Encyclopedia Brown: Case of the Midnight Visitor_](http://brownencyclopedia.tumblr.com/post/26499155951/the-case-of-the-midnight-visitor)! I loved that series as a kid.

Reply:
Haha that series reminds me of Detective Conan, which I grew up on.
[Youtube: The more famous Conan making fun of Detective Conan](https://www.youtube.com/watch?v=EstNcJOQRzI)